application/models/User_Consumption_LoginRole.php

```diff
@@ -20,8 +21,10 @@ protected function dependencies()
             Login::class,
             User::class,
             Consumption::class,
+			Transaction::class,
```

Comment:
Tabs vs. spaces difference it seems.

Reply:
![image](https://user-images.githubusercontent.com/15435678/116194719-e1e83c80-a739-11eb-8264-97e58bdd1e8e.png)